### PR TITLE
xio: sync work

### PR DESCRIPTION
**Pull request by vuhuong**
. Revert XioConnection features back to XIO_ALL_FEATURES for now
. Handling connection error event 
. Configurable max send inline
. Reduce default depth of accelio msg queue
. Reduce quantum to grow pool allocators

### CODE REVIEW

File: ceph-object-corpus [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 7bd9bbe235c59527dd1ab192fed21944da686d20
+Subproject commit da43eb119558fb46d86e62f3b82d5ca63546f69e
```
Comment (tchaikov) [inline ceph-object-corpus]: @vuhuong, is this change included by accident?

Comment (vuhuong) [inline ceph-object-corpus]: @tchaikov,

Yes, it's a mistake, my bad.

**Comment by vuhuong** [comment]
@tchaikov -  I fixed the wrong change in ceph-object-corpus

**Comment by tchaikov** [comment]
@vuhuong could you please squash commit e1faac1 with  a60d103, i.e. to squash the fix with the one which introduces the build failure?

**Comment by vuhuong** [comment]
@tchaikov - I squashed these two commits.
Sorry for double pushes of same commits but I hope they are ok.

**Comment by vuhuong** [comment]
@tchaikov - any update on this? Do I need to do anything else?

**Comment by tchaikov** [comment]
> @vuhuong sorry for the latency, could you please rebase & repush , to poke the built bot again?
> 
> once the build completes, will get it merged.
> 
> thanks,

i merged it by accident. will revert it if any of the gitbuilders yells at me.

sorry, i should not take any beer when checking the pull requests!

**Pull request merged**